### PR TITLE
Responsive layout for customer area

### PR DESCRIPTION
**Pull request by ishcherbakov**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Responsive layout - Customer area |
| Type? | new feature |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | - |
| How to test? | - |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

**Comment by maximebiloe** [comment]
Thank you!
@AlexEven could you check the domains, please?

**Review by AlexEven — CHANGES_REQUESTED** [review]
Thanks Ivan!

**Comment by ishcherbakov** [comment]
@AlexEven Done !

File: themes/classic/assets/js/theme.js [hunk 1]
```diff
@@ -11794,12 +11780,12 @@
 /***/ function(module, exports) {
 
 	/*!
-	 * Bootstrap v4.0.0-alpha.4 (http://getbootstrap.com)
+	 * Bootstrap v4.0.0-alpha.3 (http://getbootstrap.com)
```
Comment (maximebiloe) [inline themes/classic/assets/js/theme.js]: You need to run `npm update` before compiling your assets. You're downgrading Bootstrap 😢

File: themes/classic/templates/customer/_partials/order-detail-return.tpl [hunk 2]
```diff
@@ -79,39 +112,116 @@
...
+                      <div class="s" id="_mobile_return_qty_{$product.id_order_detail}_{$customization.id_customization}"></div>
+                    {/foreach}
+                  {else}
+                    <div class="q">{l s='Quantity' d='Shop.Theme.Catalog'}: {$product.quantity}</div>
+                    {if $product.quantity > $product.qty_returned}
+                      <div class="s" id="_mobile_return_qty_{$product.id_order_detail}"></div>
+                    {/if}
+                  {/if}
+                  {if $product.qty_returned > 0}
+                    <div>{l s='Returned' d='Shop.Theme.Catalog'}: {$product.qty_returned}</div>
```
Comment (maximebiloe) [inline themes/classic/templates/customer/_partials/order-detail-return.tpl]: @ishcherbakov You need to update the domain here

**Review by maximebiloe — CHANGES_REQUESTED** [review]

**Review by maximebiloe — APPROVED** [review]

**Review by AlexEven — APPROVED** [review]

**Comment by maximebiloe** [comment]
Thank you!

**Pull request merged**